routes/api-routes.js

```diff
@@ -2,9 +2,19 @@ const router = require('express').Router();
 
 // import controllers here...
 const postsController = require('../controllers/postsController.js');
+const usersController = require('../controllers/usersController.js');
 
 // routes and route handlers (controllers)
+
+//for the posts
 router.get('/getPosts', postsController.getPosts);       // get all posts
 router.get('/viewPost/:pid', postsController.viewPost);  // get single post
-// export
+router.get('/getGallery', postsController.getGallery);
```

Comment:
this is the same with /getPosts

technically, you're not getting the gallery, you are getting the posts. It's confusing when it's named getGallery (in a data perspective)

Reply:
Nevermind, I renamed it before kaya pala na push din dito. Merge conflicts hehe